project/Dependencies.scala

```diff
@@ -11,7 +11,7 @@ object Dependencies {
 
   val workbenchUtilV    = "0.3-0e9d080"
   val workbenchModelV   = "0.11-2bddd5b"
-  val workbenchGoogleV  = "0.16-7c48765"
+  val workbenchGoogleV  = "0.16-ae1b3c9-SNAP"
```

Comment:
Will change to non-`SNAP` once https://github.com/broadinstitute/workbench-libs/pull/166 is merged